### ISSUE TITLE
Continuous Deployment

### ISSUE THREAD

**Issue by hhsnopek** [carrot/roots-www#6]
It would be cool if we used Netlify or BitBallon for continuous deployment :smile:

**Comment by jenius** [comment]
I guess, I considered this but didn't see an immediate need for it

**Comment by hhsnopek** [comment]
Yeah, definitely a low priority - just thought I'd make an issue for it for the future
